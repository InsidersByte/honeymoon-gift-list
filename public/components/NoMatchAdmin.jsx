import React from 'react';
import { Link } from 'react-router';

export default function NoMatchAdmin() {
    return (
        <div>
            <h1>Page not found</h1>

            <p>Woops! Sorry, there is nothing to see here.</p>

            <p><Link to="/admin">Back to Admin</Link></p>
        </div>
    );
}
