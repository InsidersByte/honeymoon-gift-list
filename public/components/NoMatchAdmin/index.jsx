/* @flow */

import React from 'react';
import { Link } from 'react-router-dom';
import { ADMIN_ROUTE } from '../../constants/routes';

const NoMatchAdmin = () => (
  <div>
    <h1>Page not found</h1>

    <p>Woops! Sorry, there is nothing to see here.</p>

    <p><Link to={ADMIN_ROUTE}>Back to Admin</Link></p>
  </div>
);

export default NoMatchAdmin;
