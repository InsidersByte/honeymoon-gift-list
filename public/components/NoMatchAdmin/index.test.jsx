import React from 'react';
import { shallow } from 'enzyme';
import NoMatchAdmin from './';

describe('NoMatchAdmin', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<NoMatchAdmin />);
        expect(wrapper).toMatchSnapshot();
    });
});
