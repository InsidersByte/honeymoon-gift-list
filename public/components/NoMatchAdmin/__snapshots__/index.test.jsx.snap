// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatchAdmin should render correctly 1`] = `
<div>
  <h1>
    Page not found
  </h1>
  <p>
    Woops! Sorry, there is nothing to see here.
  </p>
  <p>
    <Link
      replace={false}
      to="/admin"
    >
      Back to Admin
    </Link>
  </p>
</div>
`;
